packages/websocket-plugin/tests/websocket.module.spec.ts

```diff
@@ -62,7 +63,22 @@ describe('NgxsWebsocketPlugin', () => {
     });
   }));
 
+  it('should dispatch WebSocketMessageError on error', fakeAsync((done: () => void) => {
```

Comment:
use the signature that Jasmine provides `done: DoneFn`

Reply:
updated